platform_it/src/main/resources/hadoop-metrics2-hbase.properties

```diff
@@ -0,0 +1,44 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
What's the point of this file? It seems fully commented out.

Reply:
This file overrides the default that has some of those lines uncommented. We could certainly have an empty file, I just left the original for reference/example if we ever need to enable some of them.